kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/NonJVMClientsST.java

```diff
@@ -68,6 +68,28 @@ void produceAndConsumeWithKcatClients(String namespace) {
                 .allSatisfy(v -> assertThat(v).contains(MESSAGE));
     }
 
+    /**
+     * Produce and consume message with python client.
+     *
+     * @param namespace the namespace
+     */
+    @Test
+    void produceAndConsumeWithPythonClients(String namespace) {
```

Comment:
I'm ok with merging this but it would nice to refactor so its a parameterised test which injects the Kafka client.